website/docs/whats-new/release-notes/partials/components.md

```diff
@@ -14,6 +14,11 @@
 
 ## 4.1.1
 
+**Caution**
```

Comment:
this file is auto-generated so the next time the `generate-changelog-markdown-files` script will run it will overwrite this markdown with the one in the original file.

this also means it will not be rendered as in GitHub but like this
<img width="1257" alt="screenshot_3610" src="https://github.com/hashicorp/design-system/assets/686239/bc8012dc-3c89-4de6-b83c-a3cc94c164a7">

Reply:
Maybe I would go with something that works for both:
```
**🚨 Caution: This version is missing the pre-compiled CSS 🚨**
```